src/main/java/com/ays/auth/config/AysTokenConfigurationParameter.java

```diff
@@ -65,15 +65,15 @@ public AysTokenConfiguration(AysParameterService parameterService) {
                 .map(Integer::valueOf)
                 .orElse(Integer.valueOf(AysConfigurationParameter.AUTH_REFRESH_TOKEN_EXPIRE_DAY.getDefaultValue()));
 
-        final String privateKeyPem = Optional
+        final String encryptedPrivateKeyPem = Optional
```

Comment:
I cannot see any methods named `AysPrivateKeyEncryptionUtil.encrypt` and `AysPublicKeyEncryptionUtil.encrypt`. Where do we use all these methods?

Reply:
These two classes come from the library I created. I didn't include the hashing part in the project to prevent it from being visible. 

> Library link: https://github.com/afet-yonetim-sistemi/ays-be-encryption-utility